internal/proto/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -197,7 +197,7 @@ message OidcAuthMethodAuthenticateCallbackResponse {
 // The structure of OIDC token request parameters
 message OidcAuthMethodAuthenticateTokenRequest {
   // The ID of the pending token
-  string token_id = 10 [json_name = "token_id"];  // @gotags: `class:"private"`
+  string token_id = 10 [json_name = "token_id"];  // @gotags: `class:"secret"`
```

Comment:
Not sure about this one, I made it `secret` because it was already `private`, but this is just an ID right?

Reply:
It's a secret.  See `Authenticate:Token` in [ICU-021](https://docs.google.com/document/d/10QU4DcfWMrYDQGJqrzExpzEA94TAbIA923Op4tVMYGo/edit)